packages/flutter/lib/src/material/slider.dart

```diff
@@ -17,8 +18,12 @@ import 'slider_theme.dart';
 import 'theme.dart';
 
 // Examples can assume:
+// int _dollars = 0.0;
 // int _duelCommandment = 1;
 
+/// A callback that formats a numeric value from a [Slider] widget.
```

Comment:
Maybe add a See also to Slider. semanticFormatterCallback which shows an example usecase?